docs/src/AlgebraicGeometry/Surfaces/duValSing.md

```diff
@@ -0,0 +1,11 @@
+
+```@meta
+CurrentModule = Oscar
+```
+
+# quick classifier/identifier for du Val singularities
```

Comment:
```suggestion
# Classifier/identifier for du Val singularities
```

Reply:
No, we also have a full classifier of singularities. This is a quick classifier on the basis of invariants.

Reply:
What about
# Classifier/identifier specifically for du Val singularities

Reply:
Is this full classifier of singularities accessible and documented somewhere?

`# Classifier/identifier specifically for du Val singularities`
Looks good to me.

Reply:
> Is this full classifier of singularities accessible and documented somewhere?

Arnold's classifier is accessible in several implementations with slightly different capabilities in Singular as a library and as such accessible. I did not find any documentation on it though.
Yep, this should be put on the todo-list and after it is documented, a crossreference needs to be set here. 

> `# Classifier/identifier specifically for du Val singularities` Looks good to me.

Then I shall change it this way.

Reply:
done